database/mssql/src/test/resources/sql-src/cleanup-call-test.sql

```diff
@@ -0,0 +1,36 @@
+DROP PROCEDURE CALL_TEST_IN_INTEGER_TYPES;
```

Comment:
license header missing

Reply:
Adding the license header would make the script execution logic break, so this was intentional.